src/content/blog/authors/dianna-ma/index.md

```diff
@@ -0,0 +1,9 @@
+---
+type: author
+title: "Dianna Ma"
+description: "Dianna is the creator of swiftinit.org and member of the Swift Server WorkGroup. She is passionate about Swift and server-side development."
```

Comment:
nit: should be *Work Group*, not WorkGroup

Reply:
I'll fix that in a next PR